citesphere/src/main/java/edu/asu/diging/citesphere/core/service/impl/CitationManager.java

```diff
@@ -430,6 +430,33 @@ public CitationResults getGroupItems(IUser user, String groupId, String collecti
 
     }
 
+    /**
+     * Get all citations that will be matched to citation author uri
+     * , editor uri or contributor uri with that of the query parameter uri.
+     *
+     * @param user    User who is accessing Zotero.
+     * @param uri     Uri is the query parameter.
+     * @param page    page is to paginate/limit the data returned.
+     * @return all matching citations.
+     *
+     */
+    @Override
+    public CitationResults getItemsByUri(IUser user, String uri, int page) {
```

Comment:
This name suggests that the items have a uri, and that URI is searched. However, that's not the case. Name should reflect what the method is actually doing.